x-pack/plugins/gis/public/components/layer_panel/settings_panel/index.js

```diff
@@ -23,7 +23,6 @@ function mapStateToProps(state = {}) {
     layerId: selectedLayer.getId(),
     maxZoom: selectedLayer.getMaxZoom(),
     minZoom: selectedLayer.getMinZoom(),
-    renderSourceDetails: selectedLayer.renderSourceDetails,
```

Comment:
should be careful for a couple small merge conflicts with https://github.com/elastic/kibana/pull/29294/files